entrypoint.sh

```diff
@@ -246,6 +247,14 @@ then
    fruit:wipe_intentionally_left_blank_rfork = yes
    fruit:delete_empty_adfiles = yes" > /etc/samba/smb.conf
   fi
+  if [ "${IGNORE_DOS_ATTRIBUTES}" == "true" ]
```

Comment:
Because we use `sh` instead of `bash`, this will need to be `=` instead of `==`.

[entrypoint.sh|250 col 35 warning| In POSIX sh, == in place of = is undefined. [SC3014]](https://github.com/koalaman/shellcheck/wiki/SC3014)

Reply:
Oh, good to know. Thanks.